README.md

```diff
@@ -1,7 +1,17 @@
 # Popular Movies
 An Android app to allow users to discover the most popular movies playing.
 
-### To Contribute
-Look at the issues and contribute accordingly.
 
 
+![screenshot](https://github.com/alouanemed/Popular-Movies/blob/master/screenshots.png)
+
+
```

Comment:
Mentioning how to add thmoviedb api key in the code will be more useful for the other developers to get started easily